src/hexer/nifcgen.nim

```diff
@@ -203,7 +203,8 @@ proc traverseTupleBody(e: var EContext; c: var Cursor) =
   while c.kind != ParRi:
     if c.substructureKind == FldU:
       inc c # skip fld
-      e.offer c.symId
+      if c.kind == SymbolDef:
+        e.offer c.symId
```

Comment:
So how is it possible that there is not a SymbolDef here?

Reply:
Sem produces ident tokens for anonymous tuple field names:

https://github.com/nim-lang/nimony/blob/00ae724fa94acf4c2b764ea64aa50be7b7c15902/src/nimony/sem.nim#L2379-L2381

https://github.com/nim-lang/nimony/blob/00ae724fa94acf4c2b764ea64aa50be7b7c15902/src/nimony/sem.nim#L4513-L4521

These could be changed to generate anonymous symbols but tuple field symbols do not really matter in typed AST anymore, tuple access is done via `tupat` and nifcgen ignores field symbols and names fields after their indices. I figured it might cause more trouble if we tried to always generate symbols.